packages/api/src/server.js

```diff
@@ -126,6 +127,7 @@ app.use(bodyParser.json())
   .use('/farmdata', userFarmDataRoute)
   .use('/user_farm', userFarmRoute)
   .use('/roles', rolesRoutes)
+  .use('', organicCertifierSurveyRoutes)
```

Comment:
I don't like the empty route here, if we need to have the get by farm lets add it to the farm routes.